css/style.css

```diff
@@ -216,3 +217,192 @@ ul {
 }
 
 
+
+
+
+/* Block about */
+.about {
+    padding: 0 375px 30px 375px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  
+}
+
+.about__title {
+    font-family: Roboto Slab, sans-serif;
+    font-size: 24px;
+    font-weight: bold;
+    margin-bottom: 60px;
+    margin-top: 100px;
+}
+
+.about__text {
+    margin-bottom: 28px;
+    font-family: Rubik, sans-serif;
+    font-size: 16px;
+}
+
+.about__box {
+    display: flex;
+    align-items: start;
+    gap: 40px;
+    margin-bottom: 150px;
+}
+
+.about__block {
+    display: inline;
+    align-items: center; 
+    text-align: center;
+}
+
+/* about__image */
+.about__image {
+    margin-bottom: 10px;
+    flex-shrink: 0;
+}
+
+.about__image-React {
+    width: 58px;
+    height: 52px;
+}
+
+.about__image-Redux {
+    width: 58px;
+    height: 55.23px;
+}
+
+.about__image-JS {
+    width: 52.63px;
+    height: 58px;
+}
+
+.about__image-Git {
+    width: 58px;
+    height: 58px;
+}
+
+.about__image-Mongo {
+    width: 25.92px;
+    height: 58px;
+}
+
+.about__image-MySQL {
+    width: 58px;
+    height: 56.82px;
+}
+
+
+
+
+
+.about__name {
+    font-family: Rubik, sans-serif;
+    font-size: 14px;
+    line-height: 20px;
+    color: #342B2B;
+    display: flex;
+    justify-content: center; 
+    text-align: center;
+}
+
+.box {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background: #EEF1F2;
+}
+
+.about__picture {
+    width: 464px;
+    height: 348px;
+    margin-left: 40px;
+    margin-top: 193px;
+}
+
+
+
+
+/* Block aside */
+.aside {
+    padding: 0 375px 30px 375px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: #FFFFFF;
+    margin-bottom: 150px;
+}
+
+.aside__title {
+    font-family: Roboto Slab, sans-serif;
+    font-size: 24px;
+    line-height: 32px;
+    font-weight: bold;
+    margin-bottom: 60px;
+}
+
+.aside__block {
+    display: flex;
+    gap: 17px;
+}
+
+.aside__group {
+   margin-top: 5px;
+}
+
+.aside__sign {
+    width: 14px;
+    height: 13.76px;
+    background: #EEF1F2;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.aside__triangle {
+    width: 3.29px;
+    height: 3.24px;
+	border-top: 3.24px solid #EEF1F2;
```

Comment:
what about alignment? Have you installed prettier?

Reply:
+